modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -73,6 +73,17 @@ REMOVE n[$prop]
 
 | Added the ability to dynamically reference properties in xref:clauses/set.adoc#set-dynamically-a-property[SET] and xref:clauses/remove.adoc#remove-remove-a-property-dynamically[REMOVE] clauses.
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+DROP [COMPOSITE] DATABASE ... [RESTRICT \| CASCADE ALIAS[ES]]
+----
+
+| Added the ability to drop any database aliases, that would otherwise block the deletion of the database, at the same time as the deletion of the database.
```

Comment:
I think it's either ', which' or 'that' but that's probably one for the docs team to arbitrate.

Reply:
changed in the alternative from docs review